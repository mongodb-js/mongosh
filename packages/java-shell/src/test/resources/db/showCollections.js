// before
use test_show_collections
// command
db.coll.insert({});
// command
show collections
// clear
db.coll.drop();
