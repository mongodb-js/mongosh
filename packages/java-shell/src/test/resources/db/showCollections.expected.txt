{ "acknowledged": true, "insertedId": <ObjectID> }
[ { "name": "coll", "badge": "" } ]